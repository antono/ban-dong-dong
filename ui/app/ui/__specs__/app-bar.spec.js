import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import AppBar from '../app-bar';

function setup(propsExtension = {}) {
  const props = Object.assign({
    title: 'Hello World',
    signedIn: true,
    handleAppBarLeftClick: jest.fn(),
  }, propsExtension);

  const component = shallow(<AppBar.WrappedComponent {...props} />);

  return { props, component };
}

describe('AppBar', () => {
  it('matches snapshot', () => {
    const { component } = setup({ signedIn: true });
    expect(toJSON(component)).toMatchSnapshot();
  });
});
