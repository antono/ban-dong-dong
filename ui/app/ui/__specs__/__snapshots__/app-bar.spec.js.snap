exports[`AppBar matches snapshot 1`] = `
<AppBar
  iconElementRight={<Connect(UserMenu) />}
  onLeftIconButtonTouchTap={[Function]}
  showMenuIconButton={true}
  title="Hello World"
  zDepth={1} />
`;
